test-data-generator/src/main/scala/za/co/absa/spline/testdatagen/GenCLI.scala

```diff
@@ -32,43 +52,56 @@ object GenCLI {
         head(
           s"""
              |Spline Test Data Generator
-             |Version: ${SplineBuildInfo.Version} (rev. ${SplineBuildInfo.Revision})
+             |Version: ${SplineBuildInfo.Version})
```

Comment:
```suggestion
             |Version: ${SplineBuildInfo.Version} (rev. ${SplineBuildInfo.Revision})
```
The message pattern has to stay consistent and reflect the VCS revision.